packages/mdc-icon-button/foundation.ts

```diff
@@ -22,37 +22,30 @@
  */
 
 import MDCFoundation from '@material/base/foundation';
-/* eslint-disable no-unused-vars */
-import {MDCIconButtonToggleAdapter, IconButtonToggleEvent} from './adapter';
+import {MDCIconButtonToggleAdapter} from './adapter';
 import {cssClasses, strings} from './constants';
 
-/**
- * @extends {MDCFoundation<!MDCIconButtonToggleAdapter>}
- */
-class MDCIconButtonToggleFoundation extends MDCFoundation {
-  static get cssClasses() {
+class MDCIconButtonToggleFoundation extends MDCFoundation<MDCIconButtonToggleAdapter> {
+  static get cssClasses(): {[key: string]: string} {
```

Comment:
Let the type be inferred 

https://github.com/material-components/material-components-web/pull/4319#discussion_r252748427